docs/sources/panels/field-configuration-options.md

```diff
@@ -0,0 +1,228 @@
+
+All [field options](#field-options) are defined below.
+
+### Standard field options
+
+Standard field options are:
+
+- [Unit](#unit)
+- [Min](#min)
+- [Max](#max)
+- [Decimals](#decimals)
+- [Display name](#display-name)
+- [No value](#no-value)
+- [Thresholds](#thresholds)
+- [Value mappings](#value-mapping)
+- [Data links](#data-links)
+
+You can apply standard field options to the following panel visualizations:
+
+- Bar gauge
+- Gauge
+- Stat
+- Table
+
+### Custom field options
+
+You can only apply custom field options to table visualizations.
+
+Custom field options are:
+
+- [Column width](#column-width)
+- [Column alignment](#column-alignment)
+- [Cell display mode](#cell-display-mode)
+
+## Configure all fields
+
+To change how all fields display data, you apply a [field option](#field-options). Usually you apply changes that you want to most or all of the fields here, than apply field overrides to exceptions.
+
+1. Navigate to the panel you want to edit, click the panel title, and then click **Edit**.
+2. Click the **Field** tab.
+3. Find the 
+4. Enter options by adding values in the fields. To return options to default values, delete the white text in the fields.
+5. When finished, click **Save** to save all panel edits to the dashboard.
+
+## Override a field
+
+Field overrides allow you to change the settings for one field (column in tables) to be different than the others. Field options for overrides are exactly the same as the field options available in a particular visualization. The only difference is that you choose which field to apply them to. 
+
+1. Navigate to the panel you want to edit, click the panel title, and then click **Edit**.
+1. Click the **Overrides** tab.
+1. Click **Add override**.
+1. Select a filter option to choose which fields the override applies to. 
+   
+   **Note:** Currently you can only match by name, so after you choose the filter, select which field it applies to in the dropdown list.
+
+1. Click **Add override property**.
+1. Select the [field option](#field-options) you want to apply.
+1. Enter options by adding values in the fields. To return options to default values, delete the white text in the fields.
+1. Continue to add overrides to this field by clicking **Add override property**, or you can click **Add override** and select a different field to add overrides to.
+1. When finished, click **Save** to save all panel edits to the dashboard.
+
+## Field options
+
+This section explains all available field options. They are listed in alphabetical order.
+
+Most field options will not affect the visualization until you click outside of the field option box you are editing.
+
+### Cell display mode
+
+This custom field option applies only to table visualizations.
```

Comment:
Shouldn't we to it like this:
`
> This custom field option applies only to table visualizations.
`

Reply:
You add notes to draw attention to something. We do not want to overuse notes, because that reduces their impact. In this case, it is fine as written.

Reply:
Should we move:
* Cell display mode
* Column alignment
* Column width

to the table documentation?  each visualization *may* have per/field settings

I think "custom" makes sense for plugin authors, but not for people using the panel

Reply:
No. We should keep the field options together for now, but I will be crosslinking the topics once everything is merged.

Reply:
The UI question, however, I agree with you. Calling them "custom" when they really aren't is weird, but that is a 7.1 thing at the earliest.